examples/Python3/demo4.py

```diff
@@ -0,0 +1,69 @@
+import snowboydecoder
+import sys
+import signal
+import speech_recognition as sr
+import os
+
+"""
+This demo file shows you how to use the new_message_callback to interact with
+the recorded audio after a keyword is spoken. It leverages the
+speech_recognition library in order to convert the recorded audio into text.
+"""
+
+
+interrupted = False
+
+
+def newMessageCallback(fname):
```

Comment:
Shall we call it audioRecorderCallback(fname)? Is there a reason why you chose newMessageCallback(fname)?

Reply:
Great idea. I think audioRecorderCallback() is a better name. I have changed it.